### DISCUSSION TITLE
On Windows 11, randomly an unknown panel is focused and NVDA reports the word fonts

### DISCUSSION THREAD

**Discussion by nvdaes** [nvaccess/nvda#17556] [Q&A]
Hello:
On Windows 11, using NVDA 2024.4.1, randomly an emergent window is focused and NVDA speaks the word fonts ("fuentes" in Spanish).
This has been also reported on this [thread of a spanish mailing list](https://nvdaes.groups.io/g/lista/topic/110201042).
Can someone think about a possible cause or workaround to fix this?
I prefer to start a discussion before opening an issue.

**Comment by nvdaes** [comment]
cc: @josephsl , @seanbudd , @SaschaCowley

**Comment by nvdaes** [comment]
Hi again. Using object navigation I have found the window that gains focus randomly. I think we should change the event_gainFocus of this window, or other properties, so it's not focused. Comments are appreciated.

```
Developer info for navigator object:
name: 'Sus paneles'
role: GROUPING
processID: 15844
roleText: None
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.UIA.UIA object at 0x0564F050>
Python class mro: (<class 'NVDAObjects.UIA.UIA'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: ''
location: RectLTWH(left=20, top=23, width=88, height=165)
value: None
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
appModule: AppModule(appModuleHandler, appName='widgetboard', processID=15844)
appModule.productName: 'MicrosoftWindows.Client.WebExperience'
appModule.productVersion: '524.30502.30.0'
appModule.helperLocalBindingHandle: None
appModule.appArchitecture: 'AMD64'
windowHandle: 131950
windowClassName: 'Microsoft.UI.Content.DesktopChildSiteBridge'
windowControlID: 0
windowStyle: 1342177280
extendedWindowStyle: 2097152
windowThreadID: 9012
windowText: ''
displayText: ''
UIAElement: <POINTER(IUIAutomationElement) ptr=0x6b6f8f0 at 565fd50>
UIA automationID: MenuItemsHost
UIA frameworkID: XAML
UIA runtimeID: (42, 131950, 4, 9)
UIA providerDescription: [pid:15844,providerId:0x0 Main(parent link):Unidentified Provider (unmanaged:Microsoft.UI.Xaml.dll)]
UIA className: 
UIA patterns available: ScrollItemPattern, LegacyIAccessiblePattern
```

**Comment by josephsl** [comment]
Hi, did the people experiencing this issue somehow press Windows+W? This shows/hides widgets. Thanks.

**Reply by nvdaes** [reply]
@josephsl No, we don't press windows+v.

**Comment by josephsl** [comment]
Hi, no, Windows+W. Thanks.

**Reply by nvdaes** [reply]
@josephsl No, we don't press windows+w. I'm sure since the windows key on my laptop is on left hand and it's difficult to press it. I don't know if this window sometimes is updated with new widgets and if in these cases it's focused.

**Comment by nvdaes** [comment]
I think that the keystroke provided by @josephsl is very relevant here. I didn't know it. Pressing windows+w, we can press down arrow until configuration options and deactivate the option Open when the mouse is over taskbar.
I have the mouse device disabled on my laptop, but I have deactivated this option hoping that this issue is fixed.
If not, I'll let you know.

**Comment by ruifontes** [comment]
Here it have solved the problem.

**Reply by seanbudd** [reply]
@nvdaes - apologies this got missed. Can you update us on the status of this issue?
Can we close this discussion and mark an answer?

**Reply by nvdaes** [reply]
@seanbudd , I think that we can close this. I no longer have this issue after deactivating widgets by pressing `windows+w`. Anyway, seems that the control presented after pressing `windows+w` is not the same now. Seems to have been modified in some Windows update.
